spec/ios_merge_strings_files_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe Fastlane do
+  describe Fastlane::FastFile do
+    let(:test_data_dir) { File.join(File.dirname(__FILE__), 'test-data', 'translations', 'ios_l10n_helper') }
+
+    def fixture(name)
+      File.join(test_data_dir, name)
+    end
+
+    describe '#ios_merge_strings_files' do
+      it 'calls the action with the proper parameters and warn and return duplicate keys' do
+        # Arrange
+        messages = []
+        allow(FastlaneCore::UI).to receive(:important) do |message|
+          messages.append(message)
+        end
+        inputs = ['Localizable-utf16.strings', 'non-latin-utf16.strings']
+
+        Dir.mktmpdir('a8c-release-toolkit-tests-') do |tmpdir|
+          inputs.each { |f| FileUtils.cp(fixture(f), tmpdir) }
+
+          # Act
+          result = Dir.chdir(tmpdir) do
+            described_class.new.parse("lane :test do
+              ios_merge_strings_files(
+                paths: ['#{inputs[0]}', '#{inputs[1]}'],
+                destination: 'output.strings'
+              )
+            end").runner.execute(:test)
```

Comment:
This technique for testing Fastlane Actions was borrowed [from how fastlane itself does it](https://github.com/fastlane/fastlane/blob/master/fastlane/spec/actions_specs/ipa_spec.rb).

It seems like a better and more recommended approach compared to directly calling `Fastlane::Actions::IosMergeStringsFilesAction.run(…)` (like we've been doing on other action specs in our code in the past), because calling `Fastlane::FastFile.new.parse` like here ensures that the spec will go thru the whole `ConfigItem` parsing and interpretation (and use the default value from the action's `ConfigItem` if any is provided, etc)